Tasks/UseDotNetV2/task.json

```diff
@@ -70,6 +67,15 @@
             "helpMarkDown": "Specify version of .NET Core SDK or runtime to install.<br/>Versions can be given in the following formats<li>2.x   => Install latest in major version.</li><li>2.2.x => Install latest in major and minor version</li><li>2.2.104 => Install exact version</li><br/>Find the value of `version` for installing SDK/Runtime, from the releases.json. The link to releases.json of that major.minor version can be found in [**releases-index file.**](https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/releases-index.json). Like link to releases.json for 2.2 version is https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/2.2/releases.json",
             "visibleRule": "useGlobalJson = false || packageType = runtime"
         },
+        {
+            "name": "vsVersion",
```

Comment:
we should update two documentations:
1. Update the readme in this doc to add this input
2. We should add a section for troubleshooting with the error people faced, and then suggest using the VS version as a fix. See [this](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-resource-group-deployment?view=azure-devops#troubleshooting) for reference

Reply:
For second one, it will not part of this PR. Talk to Nikitha (PM) with section content.